data-plane/pom.xml

```diff
@@ -32,7 +32,7 @@
 
     <!-- Maven plugins -->
     <maven.compiler.plugin.version>3.8.1</maven.compiler.plugin.version>
-    <maven.surefire.plugin.version>3.0.0-M5</maven.surefire.plugin.version>
+    <maven.surefire.plugin.version>2.22.2</maven.surefire.plugin.version>
```

Comment:
@pierDipi I noticed here https://github.com/vert-x3/vertx-junit5/commit/1dd7eb3071e346c172d4a59045bf96e8ff1d1bdf

that @jponge did similar there ... :see_no_evil: